pkg/daemon/ceph/cleanup/misc_cleanup.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cleanup
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/rook/rook/pkg/operator/ceph/cluster/mon"
+)
+
+// StartMiscCleanup is the main entrypoint function to clean up dataDirHostPath and monitor store
+func StartMiscCleanup(namespaceDir, dataDirHostPath, monSecret string) {
```

Comment:
Why name it "misc" cleanup if it's just for the host path?

Reply:
Yes, I got confused by the cleanup of sub dirs but it's essential dataDirHostPath